.idea/.name

```diff
@@ -1 +1 @@
-DesignCompose
\ No newline at end of file
+color.rs
```

Comment:
What's this do?

Reply:
Thanks, that snuck in when I tried to open up the project in Android Studio in a weird way yesterday. Guess the .gitignore didn't cover it.